components/combobox/index.css

```diff
@@ -213,6 +220,27 @@
 	.spectrum-Popover.is-open {
 		transform: translateY(var(--mod-combobox-spacing-edge-to-menu, var(--spectrum-combobox-spacing-edge-to-menu)));
 	}
+
+	&.is-readOnly .spectrum-Combobox-textfield {
+		input:read-only {
+			background-color: var(--spectrum-combobox-readonly-input-background-color);
+			border-color: var(--spectrum-combobox-readonly-input-border-color);
+
+			&:hover {
+				background-color: revert;
+			}
+		}
+
+		&.is-invalid input:read-only {
+			border-color: var(--highcontrast-textfield-border-color-invalid-default, var(--mod-textfield-border-color-invalid-default, var(--spectrum-combobox-readonly-border-color-invalid-default)));
+		}
+
+		&.is-disabled input:read-only {
```

Comment:
Feel free to get a second opinion, but I definitely think we should be more specific with `.spectrum-Combobox-input:read-only`, as opposed to `input:read-only`.